tigergraph/benchmark_on_cluster/setup.sh

```diff
@@ -19,13 +19,15 @@ else
   echo "Require apt or yum"
   exit 0
 fi
-
 sudo $installer -y update
-sudo $installer -y install net-tools sshpass parallel git gzip python3-pip
 sudo $installer -y install wget git
+sudo $installer -y install net-tools sshpass parallel git gzip python3-pip
+wget https://archives.fedoraproject.org/pub/archive/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
could you put them in a if clause that only run for amazon-linux and include comments